### PR TITLE
Feature/unit testing for build hyperlink fn

### PR DESCRIPTION
**Pull request by patrickjfl**
### What

Adds jest config and testing for buildHyperlink function.

### How to review

Observe the config, observe the testing, observe the passing of the tests.

### Who can review

You can review.

### CODE REVIEW

File: src/helpers/buildHyperlinkHelper.ts [hunk 1]
```diff
@@ -1,22 +1,29 @@
...
-        }
-        const paramsArr = [args.topic, args.variable, args.classification, args.category]
-        url = "/2021"
-        paramsArr.forEach((param) => {
-            if (param){
-                url = `${url}/${param}`
-            }
-        })
+export const buildHyperlink = (args: {
+  topic?: string;
```
Comment (VivianAllen) [inline src/helpers/buildHyperlinkHelper.ts:5]: what does the ? here do? Is this an optional arg?

Comment (patrickjfl) [reply src/helpers/buildHyperlinkHelper.ts:5]: yes exactly, it's how you specify an optional argument in typescript

File: src/helpers/buildHyperlinkHelper.ts.test.ts [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+import type { SelectedGeographyData } from "../types";
+
+const mockSelectedGeography = {
+  geoType: "lad",
+  displayName: "Lad name",
+  geoCode: "ladCode",
+  variableData: {
+    catCode: { count: 0, percentage: 0, total: 0 },
+  },
+} as SelectedGeographyData;
```
[the remark this answers is not in the document]
Comment (patrickjfl) [reply src/helpers/buildHyperlinkHelper.ts.test.ts:11]: I don't think so.  I might be wrong but I don't think writing "as SelectedGeographyData" has any impact on functionality - something's either an instantiation of a type or it isn't.  I assume it's another check that Typescript runs at compile time, to check whether your variable is actually instantiating the type you say it is. I mainly put it there for clarity/completeness.

File: src/helpers/buildHyperlinkHelper.ts (hunk 1 above)
Comment (VivianAllen) [reply src/helpers/buildHyperlinkHelper.ts:5]: wicked, thanks!

**Review by VivianAllen — APPROVED** [review]
very nice!

**Pull request merged**